signature-envelope-jws.md

```diff
@@ -52,20 +53,19 @@ An example of the claim is described below
 
 The payload contains the subject manifest and other attributes that have to be integrity protected.
 
-- **`subject`**(*descriptor*): A REQUIRED top-level node consisting of the manifest that needs to be integrity protected.
-  Please refer [Payload](#payload) section for more details.
-- **`iat`**(*number*): The REQUIRED property Issued-at(`iat`) identifies the time at which the signature was issued.
-- **`exp`**(*number*): This OPTIONAL property contains the expiration time on or after which the signature must not be considered valid.
+- **`subject`**(*descriptor*): A REQUIRED top-level property consisting of the manifest that needs to be integrity protected.
+  Please refer to the [Payload](#payload) section for more details.
+- **`iat`**(*number*): The REQUIRED property Issued-at(`iat`) identifies the time at which the signature was issued. See RFC 7519 for details.
+- **`exp`**(*number*): This OPTIONAL property contains the expiration time on or after which the signature must not be considered valid. See RFC 7519 for details.
 
-To leverage JWS claims validation functionality already provided by libraries, we have defined `iat`, `exp` as top-level nodes.
+To leverage JWS claims validation functionality already provided by libraries, we have defined `iat`, `exp` as top-level properties.
 
 **ProtectedHeaders**: Notary v2 supports only three protected headers: `alg`, `cty`, and `crit`.
 
 ```jsonc
 {
-    "alg": "RS256",
-    "cty": "application/vnd.cncf.notary.v2.jws.v1",
-    "crit":["cty"]
```

Comment:
Is there any justification for removing `crit`?

Reply:
See https://github.com/notaryproject/notaryproject/pull/148#discussion_r857016575

Reply:
Thanks for reference. Similar stuff is also defined for COSE. /cc @gokarnm